### PR TITLE
bump version 5.2.0

### PR DESCRIPTION
**Pull request by Taym95**

### CODE REVIEW

File: packages/flutter_reactive_ble/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+## 5.2.0
+
+* Bumps to Dart 2.17 minimum and upgrades melos to 3.1.0 in #762
```
Comment (werediver) [inline packages/flutter_reactive_ble/CHANGELOG.md:3]: The other lines are in imperative form (e.g. "Update...", "Bump..."). Consider making this line consistent with the others.

File: packages/flutter_reactive_ble/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
...
+* swiftlint config and inital formatting pass in #765
+* Cancel subscription when a disconnect event has been thrown in #769
+* Fix typos in #778
+* Update CI config to use Xcode 14 in #786
+* Support multiple services or characteristics with the same id in #776
+-   Breaking change: If a device has multiple characteristics with the same id, `readCharacteristic`, `writeCharacteristic` and `subscribeToCharacteristic` used to select the first of those characteristics. Now they will fail in this case. Use `resolve` or `getDiscoveredServices` instead.
```
Comment (werediver) [inline packages/flutter_reactive_ble/CHANGELOG.md:9]: Make the list bullet character consistent with the other items (`*`), remove weird spaces in the beginning of the line.

Prefer to spell ID in capital letters (or in full).

File: packages/flutter_reactive_ble/pubspec.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 name: flutter_reactive_ble
 description: Reactive Bluetooth Low Energy (BLE) plugin that can communicate with multiple devices
-version: 5.1.1
+version: 5.2.0
```
Comment (werediver) [inline packages/flutter_reactive_ble/pubspec.yaml:3]: Should we make any changes to any Podspec files?

**Review by werediver — APPROVED** [review]

File: packages/flutter_reactive_ble/pubspec.yaml (hunk 3 above)
Comment (Taym95) [reply packages/flutter_reactive_ble/pubspec.yaml:3]: Podspec.lock were removed for library packages

Comment (werediver) [reply packages/flutter_reactive_ble/pubspec.yaml:3]: Well, the only one out there doesn't seem to have ever been changed. It's probably okay as is.

(packages/reactive_ble_mobile/ios/reactive_ble_mobile.podspec)

**Pull request merged**